README.MD

```diff
@@ -21,24 +21,43 @@ To contribute to this project follow the steps below. Please ensure you are foll
 - Android SDK setup/ installed w/ Xamarin (https://docs.microsoft.com/en-us/xamarin/android/get-started/installation/android-sdk)
 - iOS SDK setup/installed w/ Xamarin (https://docs.microsoft.com/en-us/xamarin/ios/get-started/installation/)
 - XCode v11+ ( Mac / iOS only)
-- Azure Storage Emulator ( Web API )
-
+- [Azure Storage Emulator](https://docs.microsoft.com/en-us/azure/storage/common/storage-use-emulator) (Web Api)
+- [Azure Storage Explorer](https://azure.microsoft.com/en-us/features/storage-explorer/) (Easy way to upload and download files (see Local Emulator Database)
+- [ngrok](https://ngrok.com/) (Api and Mobile communication)
 
 ## The F5 Experience
 
 ### Web API
-1. Get the source code
-`git clone https://github.com/SSWConsulting/SSW.Consulting.git`
+1. Clone the repository `git clone https://github.com/SSWConsulting/SSW.Rewards.git` 
+1. Open the `API` Project within this repo 
 1. Grab connection strings from an existing dev to create local `secrets.json` file (https://docs.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-3.0&tabs=windows)
-1. Run Azure Storage Emulator
+1. Install / Run `Azure Storage Emulator` (Run the .exe or use the "start" command)
+1. Install / Run [Azure Storage Explorer](https://azure.microsoft.com/en-us/features/storage-explorer/#overview)
+1. Start Explorer (**On First Install** - See steps below)
+    1. Sign in using the "Subscription" option 
+    1. Select the "Azure" option
+    1. Next (Sign in with your account that has access to ssw1 DevOps) 
+1. Inside the Explorer - **Local & Attached** | **Storage Accounts** | **Emulator** | **Blob Containers**
+1. Create a **new container** called `profile`
+1. Copy over all files in `SeedData/profiles` (images + .xlsx) into the newly created `profile` container
+1. **(Optional)** - Remember to change/update your `SqlConnectionString` (Remove Trusted_Connection if you want to use a User/Password) 
+    `"Server=localhost;Database=SSWRewards;Trusted_Connection=True;"` 
```

Comment:
I would add this setting into the appsettings.json as it should be the default and no need to mention it in the f5